code/modules/research/designs/designs_organs.dm

```diff
@@ -0,0 +1,58 @@
+datum/design/item/mechfab/prosthetic_organ/Fabricate()
```

Comment:
Please use a leading `/` in top-level paths like these (ones with no tabs before them). It makes it much easier to search in some cases.